docs: Studio setup to include `&studio_unlock` (#2527)
zmkfirmware/zmk: docs/docs/development/hardware-integration/studio-setup.md

```

Notice how the outer column positions in the 6-column layout are mapped to positions 36, 37, etc. in the 5-column layout. The 5-column layout only uses key positions up to 35, so those bindings in the outer columns will get migrated into the "extra space" that is ignored by the smaller layout, preserved to get mapped back in place when the user switches back.

> # Studio Lock Keymap Update

Reviewer: ```suggestion
# Studio Unlock Keymap Update
```

> The stock keymap should be updated to include some ability to unlock the device for studio updates, by adding `&studio_unlock` to the

Reviewer: ```suggestion
The stock keymap should be updated to include a way to unlock the device for studio updates, by adding a [`&studio_unlock`](../../keymaps/behaviors/studio-unlock.md) binding to the
```
